src/Header.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
```

Comment:
this is not necessary ?

Reply:
My branch was a bit out of date, so it remained some old files that's already deleted from the latest dev branch. The merge from Dev didn't remove such files from my branch. I have removed them manually.